packages/core/src/_test/setup.ts

```diff
@@ -45,11 +46,13 @@ declare module "vitest" {
 }
 
 beforeEach((context) => {
+  const options = buildOptions({ cliOptions: { configFile: "", rootDir: "" } });
   context.common = {
-    options: buildOptions({ cliOptions: { configFile: "", rootDir: "" } }),
+    options,
     logger: new LoggerService({ level: "silent" }),
     errors: new UserErrorService(),
     metrics: new MetricsService(),
+    telemetry: new TelemetryService(options),
```

Comment:
Hmm. Once you have the tests running, I'm curious where the config.json will be generated during tests.